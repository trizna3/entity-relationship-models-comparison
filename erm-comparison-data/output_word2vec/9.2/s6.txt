games-weather;tile;fuel types
tile type-EMPTY
units-units
tile-tile
fuel type-fuel types
boards-board
weather-weather
REBIND_NARY_ASSOCIATION:S-weather-tile-fuel types
