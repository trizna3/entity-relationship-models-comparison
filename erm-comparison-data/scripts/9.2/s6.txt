entity set board {
	"rows"
	"columns"
};

entity set tile {
	"coordinates"
	"type"
};

entity set weather {
	"type"
};

entity set units {
	"name"
	"actual health"
	"attack strength"
	"defense strength"
};

entity set "fuel types" {
	"type"
};

association between 
{
	board MANY
	weather ONE
}{};

association between 
{
	board ONE
	tile MANY
}{};

association between 
{
	units ONE
	"fuel types" MANY
}{};

association between 
{
	weather ONE
	tile ONE
	"fuel types" ONE
}{};
