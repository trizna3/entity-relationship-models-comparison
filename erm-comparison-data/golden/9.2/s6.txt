games-EMPTY
boards-board
tile-tile
tile type-type
weather-weather
fuel type-fuel types
units-units
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-type-tile type